no-computer.md

```diff
@@ -11,41 +11,92 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My language is for people who want to tell a story, but don't have the time or ability to write down everything that's in their head. It will allow a creator to organize many little stories so that a common story emerges from the collection. I believe, with Bret Victor, that there are many people with ideas trapped in their heads, and that too often these ideas come out "stillborn or stunted".
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+How else would you tell a story? All I want to do is change the granularity of the language used, from English words to mini stories. A language also makes sense here because it presents a great opportunity to design an extensible language: users can write and add their own mini stories to the set, and they can share their extensions with the community.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Funnily enough, the night before writing this response I was contemplating the
+idea of telling a story through a series of short, unrelated anecdotes. I find
+it fascinating that a deep character can be developed just by giving away brief
+flashes of their life, and I like the way the selection of which stories to
+tell has roughly as much impact on the big story as the content of the small
+stories does. But as interesting as this idea seems, I wondered when I would
+have time to try it.
+
+Today, as I was rewatching Victor's talk, desperately hoping that his ideas
+would stir something up in me, it just clicked. Victor perfectly describes the
+tragedy of lost ideas, and I realized that my idea could easily become one of
+them, just because of lack of time. And worse -- how many people with similar
+ideas can't get their ideas out? How many stories will be lost? Anything that
+might prevent that from happening excites me.
```

Comment:
Stories recursively defined as smaller stories is cool! Also, I hate when my ideas are lost forever. A DSL for this would be awesome!